api/test/fixtures/resource_versions.yaml

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 - id: 1
-  version: 0.1
+  version: "0.1"
```

Comment:
Is it the change in the gorm to change it to string or is it because we want to return it as a string in response???

Reply:
@PuneetPunamiya , yes it has to be a string to represent "0.1.1" or even "alpha" etc